OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -117,6 +138,34 @@ public ClipboardTechSupport(TinyIoCContainer container)
                 }
                 settings.Add(new List<string> { "Screen Mode", screenMode });
 
+                try
+                {
+                    actWi = WindowsIdentity.GetCurrent();
+                    if (actWi.Owner.IsWellKnown(WellKnownSidType.BuiltinAdministratorsSid))
+                    {
+                        actIsAdmin = "Elevated (Admin)";
+                    }
+                    else
+                    {
+                        actIsAdmin = "Not Elevated";
+                    }
+                }
+                catch (Exception)
```

Comment:
Should we be logging these exceptions somewhere just in case?

Reply:
Don't have VS fully up on the new machine yet, but iirc, I don't think this class calls the logger.  If we wanted to avoid having to do that just for this case, we could probably check to see if it's an access-denied exception (in which case, we can reasonably infer it's an elevation-mismatch), and if it's some other exception, just add it to `warnings`.  Does that work?